src/v2/models/AppState.js

```diff
@@ -44,14 +44,24 @@ export default Model.extend({
         return factor.factorType;
       },
     },
-    isTerminalState: {
-      deps: ['terminal'],
-      fn: function (terminal) {
-        return !_.isEmpty(terminal);
+    isAuthenticateStep: {
+      deps: ['currentState'],
+      fn: function (currentState = {}) {
+        return currentState && currentState.step === 'AUTHENTICATE';
+      },
+    },
+    isIdentifyStep: {
```

Comment:
`isIdentifyStep` isn't used. remove it?

Reply:
It will be used to remove the sign out link on login page.